src/inmanta/db/versions/v202511030.py

```diff
@@ -0,0 +1,32 @@
+"""
+Copyright 2025 Inmanta
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Contact: code@inmanta.com
+"""
+
+from asyncpg import Connection
+
+
+async def update(connection: Connection) -> None:
+    """
+    Update index on resource table
+    """
+    schema = """
+    DROP INDEX resource_resource_id_index;
+
+    CREATE INDEX resource_environment_resource_id_index
+        ON public.resource (environment, resource_id);
```

Comment:
This index is not required. This information is already contained in other indexes, for example `resource_env_resourceid_index`.